doc/articles/concepts/overview/why-uno-platform.md

```diff
@@ -0,0 +1,275 @@
+---
+uid: Uno.Overview.WhyUno
+---
+
+# Why use Uno Platform for your project?
+
+Thank you for considering Uno Platform for your next project. While we always recommend trying out any stack before deciding on your next project, here are some tips you can use for your in-depth evaluation.
+
+## It's a platform, not just a UI framework.
+
+Uno Platform is a developer productivity **platform** and much **more than a UI framework**, for building single codebase, native, mobile, web, desktop and embedded apps with .NET.
+
+At the core is a cross-platform .NET UI framework, that allows apps to run everywhere with a single codebase.
+
+**However, built on top of this foundation is also a rich platform which includes libraries, extensions and tools that accelerate the design, development, and testing of cross-platform applications.**
```

Comment:
Bold for a whole long sentence feels odd IMO

```suggestion
However, built on top of this foundation is also a rich platform which includes libraries, extensions and tools that accelerate the design, development, and testing of cross-platform applications.
```

Reply:
@Youssef1313 I certainly agree ;)

I was cautious in this draft with certain changes because @sasakrsmanovic gave me a PDF with content when I took up this item, so I'll defer to his expertise about those formatting details. Otherwise, your proposed changes make more sense to me